documentation/src/main/asciidoc/topics/code_examples/Language.java

```diff
@@ -0,0 +1,15 @@
+import org.infinispan.protostream.annotations.ProtoEnumValue;
+
+public enum Language {
+  @ProtoEnumValue(name = "EN")
+  ENGLISH,
+  @ProtoEnumValue(number = 1, name = "DE")
+  GERMAN,
+  @ProtoEnumValue(number = 2, name = "IT")
+  ITALIAN,
+  @ProtoEnumValue(number = 3, name = "E")
```

Comment:
E or ES ?

Reply:
ES. I've changed.

Reply:
@wfink If we change E -> ES in both Language.java and Language.proto is that OK? or should neither change?

Reply:
that's ok  I just took the shortcuts I had on top of my head.
I just want  to show that it must not the full enum value

Reply:
thanks @wfink I'll change both to "ES". I think it fits a little better to have a two-letter abbreviation for each language.